src/content/docs/installation/boot_managers.md

```diff
@@ -66,6 +66,21 @@ The following is a list of its main pros and cons.
 ### Recommendation
 GRUB is the only available boot loader that supports BIOS booting. It is also the only boot manager that supports boot partition encryption (Different from disk encryption).
 
+## Limine
+Limine is a modern, advanced, and portable multiprotocol bootloader. It serves as the reference implementation for the Limine boot protocol and supports booting Linux as well as chainloading other boot loaders.
+
+### Pros
+- Supports multiple boot protocols, including Multiboot2 and the Linux boot protocols.
+- It can boot on both UEFI and BIOS systems, making it versatile for different hardware configurations.
+- Has theming similar to GRUB and direct support for Btrfs snapshots, which is enabled by default for installations using Btrfs as a filesystem.
+
+### Cons
+- Only supports a few filesystems, such as FAT12, FAT16, FAT32, and ISO9660, which may require additional setup for systems using other filesystems.
+- Unlike some other boot loaders, Limine does not automatically add an entry to the NVRAM on UEFI systems; this must be done manually using tools like `efibootmgr`.
```

Comment:
This can be handled via limine-entry-tool, which is preinstalled out of the box at us